<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings</class>
 <widget class="QWidget" name="settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QFormLayout" name="camera_form_layout">
     <item row="0" column="0">
      <widget class="QLabel" name="login_label">
       <property name="text">
        <string>Логин камеры</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="login_field">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="password_label">
       <property name="text">
        <string>Пароль камеры</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="password_field">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="ip_label">
       <property name="text">
        <string>IP адрес камеры</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="ip_field">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="port_label">
       <property name="text">
        <string>Порт камеры</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="port_field">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="stream_type_label">
       <property name="text">
        <string>Поток камеры</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="stream_type_combo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="modbus_form_layout">
     <item row="0" column="0">
      <widget class="QLabel" name="serial_port_label">
       <property name="text">
        <string>COM-порт</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="serial_port_combo"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="baudrate_label">
       <property name="text">
        <string>Скорость</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="baudrate_combo"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="bytesize_label">
       <property name="text">
        <string>Байтсайз</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="bytesize_spinbox"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="pairity_label">
       <property name="text">
        <string>Парность</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="pairity_combo"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="stopbits_label">
       <property name="text">
        <string>Стоп-бит</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="stopbits_spinbox"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="slave_id_label">
       <property name="text">
        <string>ID устройства</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="slave_id_spinbox"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
